package.json

```diff
@@ -16,6 +16,7 @@
     "dependencies": {
         "@floating-ui/dom": "^1.5.1",
         "js-cookie": "^3.0.5",
+        "linkify-html": "^4.1.3",
```

Comment:
Do I need to do something other than `composer install` to get this? `sail npm run build` fails:
```
[vite]: Rollup failed to resolve import "linkify-html" from "/var/www/html/resources/js/alpine/linkifyDirective/linkifyDirective.ts".
```

Reply:
Try running `npm install` in the RAWeb directory (or possibly `sail npm install`)